playbook/app/pb_kits/playbook/pb_title/_title.tsx

```diff
@@ -10,7 +10,7 @@ type TitleProps = {
   color?: "default" | "light" | "lighter" | "success" | "error" | "link",
   data?: {[key: string]: string},
   id?: string,
-  size?: 1 | 2 | 3 | 4,
+  size?: number | string,
```

Comment:
The only thing about changing this to `number` instead of using an enum is that [we only support 1-4](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_title/_title.tsx#L13) currently in production. So when someone passes in > 4 there won't be any applicable styles.

Reply:
The only reason I changed it was because the date_stacked.tsx file wasn't liking it.  Without the change, I was getting 2 separate errors:
<img width="744" alt="Screen Shot 2022-09-14 at 3 43 58 PM" src="https://user-images.githubusercontent.com/73710701/190248090-29aa8cdc-c327-4732-9a36-03a135a0b256.png">

<img width="710" alt="Screen Shot 2022-09-14 at 3 44 10 PM" src="https://user-images.githubusercontent.com/73710701/190248101-0014a7e2-e2c4-4413-81f3-f2070ec8afa0.png">

If I added the string versions of the numbers to title.tsx , I still got this error:
<img width="704" alt="Screen Shot 2022-09-14 at 3 45 07 PM" src="https://user-images.githubusercontent.com/73710701/190248295-a98b39a1-07b9-4435-ad92-673827fec443.png">
 
The only workaround that preserves the enum in Title is changing it to:
<img width="455" alt="Screen Shot 2022-09-14 at 3 49 52 PM" src="https://user-images.githubusercontent.com/73710701/190248869-8c801741-e06d-4644-857e-186c9401e04f.png">

and also typing 'sizes' in date_stacked.tsx like this:
<img width="351" alt="Screen Shot 2022-09-14 at 3 51 37 PM" src="https://user-images.githubusercontent.com/73710701/190249190-d4cdaa3e-d4fe-41bb-9714-30ea1b59a2cf.png">

Do you think this fix would be preferable to the way I currently have it @thestephenmarshall ? Let me know and I can make that change!

Reply:
I think so!

Reply:
Done!